packages/libs/Dockerfile

```diff
@@ -1,18 +1,39 @@
-FROM node:18.17 as builder
-RUN apt-get install make
+FROM node:18.17-alpine as base
 
-WORKDIR /app
-COPY package*.json ./
-RUN --mount=type=cache,target=/root/.npm npm install --loglevel verbose
+# PRUNE DEPENDENCIES
+FROM base AS builder
+
+RUN apk add --no-cache libc6-compat
+RUN apk update
 
-FROM node:18.17-alpine
+WORKDIR /app
+RUN npm install turbo --global
 
-WORKDIR /usr/src/app
-COPY --from=builder /app/node_modules ./node_modules
 COPY . .
+RUN turbo prune --scope=@audius/sdk --docker
+
+# INSTALL & BUILD
+FROM base AS installer
+
+WORKDIR /app
 
-RUN npm run typecheck
-RUN npm run build
+RUN apk add --no-cache python3 py3-pip make g++ bash libc6-compat
+RUN apk update
+
+COPY .gitignore .gitignore
+COPY --from=builder /app/out/json/ .
+COPY --from=builder /app/out/package-lock.json ./package-lock.json
+COPY --from=builder /app/scripts ./scripts
+
+RUN CI=true npm run install-packages
```

Comment:
so is this Dockerfile only used for CI? like with local dev it just uses local files, and then CI simulates some kind of more prod-like build? i would think test-audius-cmd uses the same logic on local and CI

Reply:
Yeah exactly, we use Dockerfile.dev locally and on CI it builds the actual image for prod. And yes `test-audius-cmd` is failing on CI right now because the dependencies don't exist. Ideally test-audius-cmd would work like test-identity-service does now, using the "prod" version of the stack. But it's a little different because it requires the whole stack instead of just one container. Looking into some approaches now